CONTRIBUTING.md

```diff
@@ -222,12 +216,9 @@ When this happens and you're still interested in contributing, please feel free
 
 ## Backporting changes
 
-Some changes need to be copied to older versions. We use the
-[backport](https://github.com/zeebe-io/backport-action) Github Action to automate this process.
-Please follow these steps to backport your changes:
+Some changes need to be copied to older versions. We use the [backport](https://github.com/zeebe-io/backport-action) Github Action to automate this process. Please follow these steps to backport your changes:
```

Comment:
🔧 Shameless brag, the backport-action has been moved (already quite some time ago) and exists under https://github.com/korthout/backport-action. We use this specifically:
```suggestion
Some changes need to be copied to older versions. We use the [backport](https://github.com/korthout/backport-action) Github Action to automate this process. Please follow these steps to backport your changes:
```